app/controllers/provider_interface/organisations_controller.rb

```diff
@@ -1,13 +1,17 @@
 module ProviderInterface
   class OrganisationsController < ProviderInterfaceController
     before_action :render_403_unless_organisation_valid_for_user, only: :show
+    before_action :require_accredited_provider_setting_permissions_flag, only: :settings
+    before_action :require_manage_users_or_manage_organisations_permission, only: :settings
 
     def index
       @manageable_providers = manageable_providers
     end
 
     def show; end
 
+    def settings; end
```

Comment:
I don't think this really belongs in the organisations controller (which is about displaying provider_relationship_permissions really). Maybe add a separate controller just for this? If we call it OrganisationSettingsController then we can use a resourceful route rather than the manual mapping

Reply:
I tried to avoid adding a controller just for this, but what you say makes sense. Will update.